App/Package.appxmanifest

```diff
@@ -0,0 +1,52 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+    xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+    xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+    xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+    xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+    IgnorableNamespaces="uap rescap">
+
+    <Identity
+        Name="925b49fc-4648-4967-b4e6-b5473061ee62"
+        Publisher="CN=dean"
```

Comment:
![Added via Giphy](https://media0.giphy.com/media/2xPS5yG3UZYDTLaJFz/giphy.gif?cid=c9487743fvc5ljjsonnjyvecotaaquj3zw8g856i9dssw4ef&ep=v1_gifs_search&rid=giphy.gif&ct=g)

Reply:
lmfao oops, I guess it defaults to the current user